docs/alincome/overview.md

```diff
@@ -17,3 +17,23 @@ debts, and then allow users to review and add or change that information.
 The ALIncome module and associated YAML files were made to handle all of these complicated
 cases, so you can focus on the details specific to your clients.
 
+If you want to jump right into the demo interview, you can install
+the [ALToolbox package](https://github.com/SuffolkLITLab/docassemble-ALTolbox)
+on your server, and run the [al_income_demo interview](https://github.com/SuffolkLITLab/docassemble-ALToolbox/blob/main/docassemble/ALToolbox/data/questions/al_income_demo.yml), by going to "https://yourserverurl.com/start/ALToolbox/al_income_demo". You can also visit out the live demo at [Suffolk's test server](https://apps-test.suffolklitlab.org/start/ALToolbox/al_income_demo).
```

Comment:
I like that we have a demo up. Maybe list that first, since people (including me) often jump to do things instead of reading through to the end. Unless there's an advantage to them doing it instead.